src/Themes/NexusMods.Themes.NexusFluentDark/Styles/UserControls/WorkspaceSystem/PanelViewStyles.axaml

```diff
@@ -0,0 +1,93 @@
+<Styles xmlns="https://github.com/avaloniaui"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:ui="clr-namespace:NexusMods.App.UI.WorkspaceSystem;assembly=NexusMods.App.UI">
+
+    <Style Selector="ui|PanelView">
+        <Style Selector="^ ItemsControl#TabContents">
+            <Setter Property="Padding" Value="0" />
+        </Style>
+
+        <Style Selector="^ Border#PanelBorder:not(.OneTab)">
+            <Setter Property="BorderThickness" Value="1" />
+            <Setter Property="BorderBrush" Value="{StaticResource ElementStrokeTranslucentWeak}"/>
+        </Style>
+
+        <Style Selector="^ Border#TabHeaderBorder">
+            <Setter Property="BorderThickness" Value="0" />
+            <Setter Property="BorderBrush" Value="{StaticResource ElementStrokeTranslucentWeak}"/>
+
+            <Style Selector="^ Border#AddTabButton1Container, ^ Border#TabHeaderAddAndScrollBorder, ^ Border#TabHeaderSideAreaBorder">
+                <Setter Property="BorderThickness" Value="0 0 0 1"/>
+                <Setter Property="BorderBrush" Value="{StaticResource ElementStrokeTranslucentWeak}" />
+            </Style>
+        </Style>
+
+        <!-- one tab -->
+        <Style Selector="^ Border#TabHeaderBorder.OneTab">
+            <Style Selector="^ ui|PanelTabHeaderView Border#Container">
+                <Setter Property="CornerRadius" Value="8 0 0 0"/>
+
+                <Setter Property="BorderBrush" Value="{StaticResource ElementStrokeTranslucentWeak}"/>
+                <Setter Property="BorderThickness" Value="0 0 1 0" />
+            </Style>
+        </Style>
+
+        <!-- multiple tabs -->
+        <Style Selector="^ Border#TabHeaderBorder:not(.OneTab)">
+            <Setter Property="CornerRadius" Value="8 8 0 0" />
+            <Setter Property="Background" Value="{StaticResource ElementBackgroundNeutralBaseBrush}" />
+
+            <Style Selector="^ ItemsControl#TabHeaders">
+                <!-- every tab -->
+                <Style Selector="^ ContentPresenter:nth-child(n) ui|PanelTabHeaderView">
```

Comment:
Do you know if it would be better to further qualify the `ContentPresenter` with a name to make style application faster instead of having to check every ContentPresenter to see if it has a `PanelTabHeaderView` child somewhere?

I honestly don't know too much on the styling application and performance.

Reply:
They have no name.

Reply:
![2024-01-23_10-40-16](https://github.com/Nexus-Mods/NexusMods.App/assets/16577545/41e915bc-a851-4e89-b9bc-f32e74fbe323)

Reply:
Ah, I see, then this is fine